<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1196</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1201</width>
     <height>601</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="mainTab">
       <attribute name="title">
        <string>Главная страница</string>
       </attribute>
       <widget class="QWidget" name="horizontalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>-10</x>
          <y>0</y>
          <width>1201</width>
          <height>571</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="mainLay"/>
       </widget>
      </widget>
      <widget class="QWidget" name="settingsTab">
       <attribute name="title">
        <string>Настройки</string>
       </attribute>
       <widget class="QWidget" name="horizontalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1191</width>
          <height>571</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="settingsLay">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string>Градации серого</string>
             </property>
             <widget class="QWidget" name="verticalLayoutWidget_2">
              <property name="geometry">
               <rect>
                <x>9</x>
                <y>19</y>
                <width>371</width>
                <height>161</height>
               </rect>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <item>
                  <widget class="QLabel" name="label_3">
                   <property name="minimumSize">
                    <size>
                     <width>130</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>Коэффициент красного:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSlider" name="sliderRed">
                   <property name="maximum">
                    <number>100</number>
                   </property>
                   <property name="value">
                    <number>30</number>
                   </property>
                   <property name="tracking">
                    <bool>false</bool>
                   </property>
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="invertedAppearance">
                    <bool>false</bool>
                   </property>
                   <property name="invertedControls">
                    <bool>false</bool>
                   </property>
                   <property name="tickPosition">
                    <enum>QSlider::TicksBelow</enum>
                   </property>
                   <property name="tickInterval">
                    <number>10</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <item>
                  <widget class="QLabel" name="label_4">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>130</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>Коэффициент зеленого:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSlider" name="sliderGreen">
                   <property name="maximum">
                    <number>100</number>
                   </property>
                   <property name="value">
                    <number>59</number>
                   </property>
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="tickPosition">
                    <enum>QSlider::TicksBelow</enum>
                   </property>
                   <property name="tickInterval">
                    <number>10</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                 <item>
                  <widget class="QLabel" name="label_5">
                   <property name="minimumSize">
                    <size>
                     <width>130</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>Коэффициент голубого</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSlider" name="sliderBlue">
                   <property name="maximum">
                    <number>100</number>
                   </property>
                   <property name="value">
                    <number>11</number>
                   </property>
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="tickPosition">
                    <enum>QSlider::TicksBelow</enum>
                   </property>
                   <property name="tickInterval">
                    <number>10</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QPushButton" name="buttonReset">
                 <property name="text">
                  <string>По умолчанию</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string>Градиентный оператор</string>
             </property>
             <widget class="QWidget" name="verticalLayoutWidget_3">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>20</y>
                <width>371</width>
                <height>151</height>
               </rect>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_5">
                 <item>
                  <widget class="QLabel" name="label_6">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_5">
                   <item>
                    <widget class="QRadioButton" name="radioButton_1">
                     <property name="text">
                      <string>Робертса</string>
                     </property>
                     <property name="checked">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QRadioButton" name="radioButton_2">
                     <property name="text">
                      <string>Превитта</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QRadioButton" name="radioButton_3">
                     <property name="text">
                      <string>Собеля</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QRadioButton" name="radioButton_4">
                     <property name="text">
                      <string>Лапласа</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QRadioButton" name="radioButton_5">
                     <property name="text">
                      <string>Метод Кирша</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_7">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_3">
             <property name="title">
              <string>Дополнительные настройки</string>
             </property>
             <widget class="QWidget" name="verticalLayoutWidget_5">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>20</y>
                <width>371</width>
                <height>151</height>
               </rect>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_6">
                 <item>
                  <widget class="QLabel" name="label_8">
                   <property name="text">
                    <string>Путь к сохранению:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="linePath">
                   <property name="text">
                    <string>Путь не выбран...</string>
                   </property>
                   <property name="readOnly">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="buttonOpenDir">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>40</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>...</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QCheckBox" name="checkGrayOnly">
                 <property name="text">
                  <string>Только градации серого</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_7">
                 <item>
                  <widget class="QCheckBox" name="checkEdge">
                   <property name="text">
                    <string>Сравнивать с пороговым значением</string>
                   </property>
                   <property name="checked">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSpinBox" name="spinEdge">
                   <property name="maximumSize">
                    <size>
                     <width>100</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="maximum">
                    <number>700</number>
                   </property>
                   <property name="value">
                    <number>50</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QPushButton" name="buttonResetExtra">
                 <property name="text">
                  <string>По умолчанию</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
